src/export/exportChannelColumn.component.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@mui/material';
+import type { AxiosError } from 'axios';
+import React from 'react';
+import { useExportData } from '../api/export';
+import type { FullChannelMetadata } from '../app.types';
+import handleOG_APIError from '../handleOG_APIError';
+
+export interface ExportChannelColumnProps {
+  open: boolean;
+  onClose: () => void;
+  channelInfo: FullChannelMetadata;
+}
+
+const ExportChannelColumn = (props: ExportChannelColumnProps) => {
+  const { open, onClose, channelInfo } = props;
+
+  const { mutateAsync: exportChannels, isPending } = useExportData();
+
+  const handleExportChannel = React.useCallback(() => {
+    exportChannels({
+      exportType: 'All Rows',
+      dataToExport: {
+        Scalars: true,
+        Images: true,
+        'Waveform CSVs': true,
+        'Waveform Images': true,
+      },
+      selectedColumn: channelInfo.systemName,
+    })
+      .then(() => {
+        onClose();
+      })
+      .catch((error: AxiosError) => {
+        handleOG_APIError(error);
+      });
+  }, [channelInfo.systemName, exportChannels, onClose]);
+
+  return (
+    <>
+      <Dialog open={open} onClose={onClose} maxWidth="lg">
+        <DialogTitle>Export Channel</DialogTitle>
+        <DialogContent>
+          Do you want to export all files in the{' '}
+          <strong data-testid="export-channel-name">
+            {channelInfo.name || channelInfo.systemName}
+          </strong>
+          ?
```

Comment:
This reads a little clunky - need to either add `column` to the end after the channel name, or rephrease to `Do you want to export all files for the channel X`